README.md

```diff
@@ -169,13 +169,13 @@ If you are building for NVIDIA's Jetson platforms (Jetson Nano, TX1, TX2, AGX Xa
 
 Common
 ```bash
-conda install numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions future six requests
+conda install numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions future six requests pkg-config libuv
```

Comment:
@mrshenli can you please verify whether this should be added to common and/or linux?  I dont recall if it was needed other than with my mac, but I went through the instructions a second time tonight and didnt see `conda install pkg-config libuv` in any obvious place

Reply:
This is only needed for MacOS and only needed when using the distributed package, as for Linux we are using TCP.  So instead of add it here, can you add a section:

----

On MacOS

```
# Add these packages if torch.distributed is needed
conda install pkg-config libuv
```